Objective-C/CBLCollection.h

```diff
@@ -268,6 +269,17 @@ extern NSString* const kCBLDefaultCollectionName;
                                                    queue: (nullable dispatch_queue_t)queue
                                                 listener: (void (^)(CBLDocumentChange*))listener;
 
+
+/**
+ Get a query index object by name.
+
+ @param name The index name.
+ @param error On return, the error if any.
+ @return QueryIndex object if index exists. If not, it will return nil.
+ */
+- (nullable CBLQueryIndex*) getIndex: (NSString*)name 
```

Comment:
For consistency with the other such as `collectionWithName`, I think it should be `indexWithName:`.